assets/javascripts/discourse/initializers/qa-edits.js

```diff
@@ -1,6 +1,8 @@
 import I18n from "I18n";
 import { withPluginApi } from "discourse/lib/plugin-api";
 
+const ORDER_BY_ACTIVITY_FILTER = "activity";
```

Comment:
Maybe just for consistency we could add a `ORDER_BY_VOTES_FILTER = "votes"` just so it's not assumed by default to be ordered by votes?

Reply:
By default, we want Q&A topics to be ordered by votes so loading the topic's URL directly should order it by votes.